app/src/i18n/locales/de/data.json

```diff
@@ -0,0 +1,104 @@
+  "disposal-solid-waste": "Entsorgung von festen Abfällen in der Stadt",
+  "biological-treatment-waste": "Biologische Behandlung von in der Stadt erzeugten Abfällen",
+  "incineration-waste": "Verbrennung und offenes Verbrennen von in der Stadt erzeugten Abfällen",
+  "waste-water": "Abwasser in der Stadt erzeugt",
+  "skip-step-button": "Diesen Schritt überspringen",
+  "save-continue-button": "Speichern und Fortfahren",
+  "updated-every": "Aktualisiert alle",
+  "month": "Monat",
+  "year": "Jahr",
+  "day": "Tag",
+  "sources": "Quellen",
+  "methodology": "Methodik",
+  "sector": "Sektor",
+  "enter-subsector-data": "Geben Sie die Daten des Teilsektors ein",
+  "value-types": "Welche Arten von Werten haben Sie?",
+  "value-types-tooltip": "Geben Sie die Emissionsfaktoren für CO2, N2O und CH4 ein. Diese Faktoren repräsentieren die Menge jedes Gases, die pro Einheit Aktivität oder verbrauchtem Brennstoff emittiert wird. Sie finden diese Werte in den Richtlinien des Emissionsinventars Ihres Landes oder von einer vertrauenswürdigen Quelle. Stellen Sie sicher, dass die Werte mit den Einheiten Ihrer Aktivitätsdaten übereinstimmen.",
+  "one-value": "Ein Wert für den Teilsektor",
+  "subcategory-values": "Werte für jede Unterkategorie",
+  "select-methodology": "Methodik auswählen",
+  "methodology-tooltip": "Wählen Sie \"Direkte Messung\", wenn Sie tatsächliche Messungen der THG-Emissionen von einer Quelle haben, die oft mit Geräten wie Messgeräten oder Sensoren erhalten werden. Wählen Sie \"Aktivitätsdaten\", wenn Sie Informationen zur Menge einer spezifischen Aktivität haben, die Emissionen erzeugt (z.B. Kraftstoffverbrauch).",
+  "activity-data": "Aktivitätsdaten",
+  "direct-measure": "Direkte Messung",
+  "fuel-combustion": "Verbrennung von Brennstoffen",
+  "grid-supplied-energy": "Netzstrom",
+  "add-data": "Daten hinzufügen",
+  "activity-data-amount": "Menge der Aktivitätsdaten",
+  "value-required": "Bitte fügen Sie einen Wert hinzu",
+  "option-required": "Bitte wählen Sie eine Option",
+  "activity-data-amount-placeholder": "Aktivitätsdatenmenge einfügen",
+  "emission-factor-type": "Emissionsfaktortyp auswählen",
+  "emission-factors-values": "Werte der Emissionsfaktoren",
+  "co2-emission-factor": "CO2-Emissionsfaktor",
+  "n2o-emission-factor": "N2O-Emissionsfaktor",
+  "ch4-emission-factor": "CH4-Emissionsfaktor",
+  "optional": "Optional",
+  "emissions-factor-details": "Mindestens ein Emissionsfaktor ist erforderlich. Leere Werte werden als 0 berechnet und müssen begründet werden.",
+  "source-reference": "Quellenreferenz",
+  "source-reference-required": "Fügen Sie eine Beschreibung der Referenzquelle und begründete Emissionsfaktoren, falls zutreffend, hinzu",
+  "source-reference-placeholder": "Geben Sie ausführlich an, aus welcher Quelle der eingegebene Wert stammt",
+  "calculations-details": "Alle Berechnungen berücksichtigen einen GWP-Wert von <1>{{co2Value}} für CO2 und {{ch4Value}} für CH4</1>.",
```

Comment:
This is outdated, there is now a link here instead of two separate numbers that are being passed in (see value in English data translation file).

Reply:
Might also need to be updated on the spanish version of the translation.